Week1/exercise/w1/index.js

```diff
@@ -1,11 +1,22 @@
-console.log('Hack your future, right?!');
+console.log("Hack your future, right?!");
 
-const header = document.querySelector('h1');
-console.log('header: ', header);
+const header = document.querySelector("h1");
+console.log("header: ", header);
 
 // EXERCISE:
-// select "change image" button (querySelector)
-// add click event listener
+const selectIdChangeImageOfButton = document.querySelector("#btn-changeImage"); // select "change image" button (querySelector)
```

Comment:
"changeImageButton" would be a better name for this variable as its the button that will change the image